website/docs/error-reference.md

```diff
@@ -1047,6 +1047,59 @@ bar(1) # error
 bar(x: 1) # ok
 ```
 
+## 7005
+
+Sorbet detected a mismatch between the declared return type for the method and
+the type of the returned value.
+
```

Comment:
(non-blocking suggestion) Might be good to throw in an ultra-obvious example to start with, something along the lines of:

```rb
sig { returns(Integer) }
def answer
  "42" # error: Expected `Integer` but found `String("42")` for method result type 
end
```

Reply:
Good idea 👍